### ISSUE TITLE
failed to achieve good results reported in the paper

### ISSUE THREAD

**Issue by zwwaaavv** [xfflzl/CIM-CIL#7]
Hello author, I have replicated your work using the FOOD101 dataset and have run the code using the dataset split provided in your paper(LFS--10--Foster). However, the results I obtained did not match the performance reported in your paper. I obtained a result of only 69.41,which is different from the reported result in the paper of 74.85.
I am wondering if this could be due to a problem with the hyperparameters or if there may be other factors at play. Thank you for your time.
And this is the structure of my data folder:

--Food101
------train
-----------class0
--------------0.jpg
--------------1.jpg
-------------- ...
-----------class1
-------------- ...
------test
-----------class0
--------------3.jpg
--------------4.jpg
-------------- ...
-----------class1
-------------- ...

**Comment by xfflzl** [comment]
Thanks for your interest in our work!

Could you please try the experiments with other `FOSTER` hyperparameters (e.g., `beta1=0.99` and `beta2=0.97`)?

**Comment by zwwaaavv** [comment]
Thanks for your reply. I ran the model with the hyperparameters you provided, β1 and β2, but it seems that the result was even lower.

2023-07-25 04:29:09,510 [trainer.py] => CNN top1 curve: [91.49, 86.99, 82.3, 77.33, 74.91, 71.95, 68.52, 65.94, 64.12, 62.86]
2023-07-25 04:29:09,511 [trainer.py] => CNN top5 curve: [99.16, 97.85, 96.65, 95.74, 94.9, 93.74, 91.14, 88.09, 85.89, 82.86]

**Comment by xfflzl** [comment]
Did you set the memory budget setting to be "fixed" rather than "growing" when running LFS experiments?

**Comment by zwwaaavv** [comment]
OMG!maybe that's the reason.I'll try again.

**Comment by xfflzl** [comment]
Hi, the results you provided have a 74.64% top1 average accuracy. It is close to the reported results.

**Comment by zwwaaavv** [comment]
Thanks for pointing that out and thanks for your time. 

Could you please provide me with the hyperparameters for other experimental settings (or are these hyperparameters the same as those used in Foster's study)? I would greatly appreciate it.

Also, have you encountered a problem during the training where the accuracy for a particular task was significantly lower at some stage? I'm not sure if this is normal.(the 90-99 is 4.96)
 CNN: {'total': 62.86, '00-09': 72.48, '10-19': 73.2, '20-29': 71.04, '30-39': 73.08, '40-49': 81.6, '50-59': 75.52, '60-69': 65.16, '70-79': 65.08, '80-89': 49.76, '90-99': 4.96, 'old': 69.41, 'new': 3.24}

**Comment by xfflzl** [comment]
Hi, `FOSTER`-related hyperparameters used in our experiments are from original `FOSTER` codes. For the low performances for some classes, you can check the training process and see if there is loss explosion problem.
